packages/grafana-ui/src/components/Alert/Alert.tsx

```diff
@@ -60,3 +70,66 @@ export const Alert: FC<AlertProps> = ({ title, buttonText, onButtonClick, onRemo
     </div>
   );
 };
+
+const getStyles = (theme: GrafanaTheme, severity: AlertVariant, outline: boolean) => {
+  const { redBase, redShade, greenBase, greenShade, blue80, blue77, white } = theme.palette;
+  const backgrounds = {
+    error: css`
+      background: linear-gradient(90deg, ${redBase}, ${redShade});
+    `,
+    warning: css`
+      background: linear-gradient(90deg, ${redBase}, ${redShade});
+    `,
+    info: css`
+      background: linear-gradient(100deg, ${blue80}, ${blue77});
+    `,
+    success: css`
+      background: linear-gradient(100deg, ${greenBase}, ${greenShade});
+    `,
+  };
+
+  return {
+    container: css`
+      z-index: ${theme.zIndex.tooltip};
+    `,
+    alert: css`
+      padding: 15px 20px;
+      margin-bottom: ${theme.spacing.xs};
+      position: relative;
+      color: ${white};
+      text-shadow: 0 1px 0 rgba(0, 0, 0, 0.2);
+      border-radius: ${theme.border.radius.md};
+      display: flex;
+      flex-direction: row;
+      align-items: center;
+      ${backgrounds[severity]}
+    `,
+    icon: css`
+      padding: 0 ${theme.spacing.md} 0 0;
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      width: 35px;
+    `,
+    title: css`
+      font-weight: ${theme.typography.weight.semibold};
+    `,
+    body: css`
+      .alert-body {
```

Comment:
this nested rule is a bit strange, where is this class defined / used?

Reply:
Good catch, that'd be removed completely, missed it while refactoring the old styles.